src/components/navbar/navbar.jsx

```diff
@@ -0,0 +1,49 @@
+import '../navbar/navbar.css';
+import search from '../../assets/images/search.png';
+import Cart from '../../assets/images/cart.png';
+import wishlist from '../../assets/images/wishlist.png';
+import logout from '../../assets/images/logout.png';
+import { Link, NavLink } from 'react-router-dom';
+
+export const Navbar = () => {
+    return (
+        <>
+            <nav className="navbar">
+                <NavLink to="/" className="link--home" > <h3 className='link--home nav__heading'>MajorSpark</h3> </NavLink>
+                <div className="search-container">
+                    <input type="search" name="searchBar" className="input__search" placeholder="search here" />
+                    <button className="btn__search"><img src={search} alt="search" className="img--icon" /></button>
+                </div>
+                <div className="customer-info">
+                    <NavLink to="/login"> <button className="btn--login">Login</button> </NavLink>
+                    <NavLink to="/wishlist">
+                        <div className="wishlist-box">
+                            <div className="badge">
+                                <img src={wishlist} alt="wishlist image" className="badge__img" />
+                                <span className="badge__text">7</span>
+                            </div>
+                        </div>
+                    </NavLink>
+                    <NavLink to="/cart">
+                        <div className="cart-box">
+                            <div className="badge">
+                                <img src={Cart} alt="Cart image" className="badge__img" />
```

Comment:
Don't write image in alt prop, empty string is fine,

Reply:
> overall look good just do some changes.

thank you i will do the changes